scripts/build.py

```diff
@@ -1137,7 +1176,11 @@ def generate_build_directory():
             TEMPLATES_CORE_DIRNAMES_TO_DIRPATHS['compiled_js_dir']
         ],
         THIRD_PARTY_GENERATED_DEV_DIR]
-    _verify_build(SOURCE_DIRS, COPY_OUTPUT_DIRS, hashes)
+    # This is required since we need to compare build/extensions and
+    # build/templates against both the compiled js dir and dev dir.
+    SOURCE_DIRS_TYPE = [
```

Comment:
Nope -- as above, find another way that reads more cleanly.

Reply:
Updated.